pyproject.toml

```diff
@@ -1,3 +1,5 @@
+api_version = "0.1.327"
+
```

Comment:
Wasn't entirely sure where to put this other than where the SDK `version` is already sitting